images/Dockerfile

```diff
@@ -39,12 +39,14 @@ ENV RUNNER_MANUALLY_TRAP_SIG=1
 ENV ACTIONS_RUNNER_PRINT_LOG_TO_STDOUT=1
 ENV ImageOS=ubuntu22
 
-RUN apt-get update -y \
-    && apt-get install -y --no-install-recommends \
-    sudo \
-    lsb-release \
+RUN apt update -y \
+    && apt install -y --no-install-recommends sudo lsb-release software-properties-common \
     && rm -rf /var/lib/apt/lists/*
```

Comment:
I'm combining these into a single line to aid readability.  (One command per line).

The net change here is to install `software-properties-common`.  (We still install `sudo` and `lsb-release`.)

`software-properties-common` is required for the new instruction on line 47, `add-apt-repository`